### DISCUSSION TITLE
Hours of Hackathon

### DISCUSSION THREAD

**Discussion by SatinderpalSingh** [Computer-Science-and-Engineering-GNDEC/Hackathon-1.0#59] [Show and tell]
It is almost 12 hours till now the Hackathon started, and some of the teams are performing well. Some are still sleeping. We wish to remind you that each team have make commits within an hour and send PR within every 6 hours. And also discuss in the forum about your projects. 

We hope that your are developing the projects on the servers provided by GNDEC, We are expecting the URLs of all such projects upto 7 AM. 

Share your URLs here in the reply to this discussion along with the email address of the team leader.

**Comment by hsrai** [comment]
Teams might be eligible for disqualification, if they fail to show progress in the form of commit every hour.

**Reply by SatinderpalSingh** [reply]
@all Take this seriously.
In addition to commit every hour, make a progress.md in your root folder of the project and update that file every hour

**Comment by palak199** [comment]
## TEAM she-coders 
Project URL [β Analyst](http://gne11.gndec.ac.in:8023)
Github Organisation URL [she-coders](https://github.com/she-coders) 
Mail id p.thapar99@gmail.com

**Comment by SuhelMehta9** [comment]
Here is our hosted mail server  
[Admin](https://surajdadral.me/admin)  
[User mails](https://surajdadral.me/SOGo)  

Username and password for accessing the mailbox  

username: test@mail.surajdadral.me  
password: test@123  

Email of the team leader: mehtasuhel@gmail.com

**Comment by Ekam-Preet-Singh** [comment]


**Reply by SatinderpalSingh** [reply]
Post your project's url also

**Comment by lakshaychopra** [comment]
Team - Cubicle-Forces
Here is the url of our Project
URL -http://gne3.gndec.ac.in

And kindly tell where we share our credentials

**Comment by raghavbansal27** [comment]
Team- Su Assassins
Github Organisation URL- https://github.com/su-assassins/Hackathon-1.0/tree/projects
Team Leader mail- raghavbansal292@gmail.com
Project link- http://103.66.206.227:3000/

**Reply by SatinderpalSingh** [reply]
post your project's url

**Reply by raghavbansal27** [reply]
Sir, we don't have credentials for the GNDEC server. I have already mailed for credentials at cc@gndec.ac.in

**Comment by PriyankaJhamb** [comment]
Team Falcon-Coders
About Project:-https://github.com/Computer-Science-and-Engineering-GNDEC/Hackathon-1.0/blob/projects/Falcon%20Coders/README.md

Server-link : https://46867a9007f2.ngrok.io

To enter into server ,login credentials are required.
So we have send login credentials at cc@gndec.ac.in

Team leader -Priyanka Jhamb
Email - priyankajhamb73@gmail.com

**Comment by SatinderpalSingh** [comment]
[Hackathon Finishing Instructions](https://computer-science-and-engineering-gndec.github.io/Hackathon-1.0/end.html)

The presentations regarding the projects will be stated at 9AM in online mode. The time slot given to every team will be 15 mins. Book your slot by entering your details in this [sheet](https://docs.google.com/spreadsheets/d/1l3QxWPWqkOeKFk9CwiJql5c1B-qHqESWCsNo2jew3BA/edit#gid=0).

**Reply by PriyankaJhamb** [reply]
Sir, Is it compulsory that presentation will be given by only team leader?

**Reply by SatinderpalSingh** [reply]
No, it will be presented by any member of team, team leader will be point of contact.

**Comment by code-stacker** [comment]
Team name : FourDirection
Team Leader: Abhishek Kumar(abhishek.srivastava16@ymail.com)
Project URL: http://gne5.gndec.ac.in:8987/
Github Repo URL: https://github.com/FourDirection/Hackathon-1.0

**Comment by lokesh-2001** [comment]
Team - TEAM-cybernated-yearning

Team Leader- Lokesh Dhingra(lokesh2001dhingra@gmail.com)

project URLl- https://17129bd2eb6d.ngrok.io/Hackathon-1.0/index.html

Github Repo URL-  https://github.com/TEAM-cybernated-yearning/Hackathon-1.0

**Comment by Mr-codehunter** [comment]
Sir hackthon 2.0 is the part of hackthon 1.0 or not???

**Reply by SatinderpalSingh** [reply]
It is Hackathon 1.1 and it is indendent of Hackathon 1.0
